manifests/params.pp

```diff
@@ -9,6 +9,7 @@
         }
         default: {
           $rubygem_katello = 'tfm-rubygem-katello'
+          $rubygem_katello_ostree ='tfm-rubygem-katello-ostree'
```

Comment:
It should also be defined for other operating systems. In the foreman modules we always set it even if it's unpackaged. That way you can package it in a minor/patch release without changing the modules.